src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -280,12 +288,11 @@ namespace stan {
             }
             i += 1;
           } catch (std::exception& e) {
-            this->write_error_msg_(print_stream, e);
             n_monte_carlo_drop += 1;
-            if (n_monte_carlo_drop >= 5*n_monte_carlo_grad) {
+            if (n_monte_carlo_drop >= 10*n_monte_carlo_grad) {
               const char* name = "The number of dropped evaluations";
               const char* msg1 = "has reached its maximum amount (";
-              int y = 5*n_monte_carlo_grad;
+              int y = 10*n_monte_carlo_grad;
```

Comment:
Rather than cut-and-paste the magic number 10 in two places, it's better to call it out as a named static constant.